src/components/ClickButton.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { CoreButtonProps } from "./Buttons";
```

Comment:
This is a circular import. I think pull the `CoreButtonProps` into a `CoreButton.ts`, and move the `export * from "..."` into this file.

Reply:
> I think pull the `CoreButtonProps` into a `CoreButton.ts`

Or even `BaseButton`.